### ISSUE TITLE
Fontconfig error

### ISSUE THREAD

**Issue by dagophil** [ilastik/ilastik#1146]
Hi,

when I run ilastik 1.1.6 (ubuntu 14.04), the terminal shows
`Starting ilastik from "/home/philip/inst/ilastik-1.1.6-Linux".`
`Fontconfig error: Cannot load default config file`
and some fonts appear non-smooth. I did not have this error in ilastik 1.1.5.
Since ilastik still works as expected, this is not really a problem, I just thought you may want to know.

Regards,
Philip

![ilastik_fonts](https://cloud.githubusercontent.com/assets/6207208/9812130/88ebbdb8-587b-11e5-84b9-f2d515a90e97.png)

**Comment by burgerdev** [comment]
Do you have the `fontconfig` package installed (does `find /usr/lib/ -name libfontconfig.so` yield a result)? This might be a linker issue.

**Comment by dagophil** [comment]
Yes the package is installed and `libfontconfig.so` can be found in `/usr/lib/x86_64-linux-gnu/libfontconfig.so` and `/home/philip/inst/ilastik-1.1.6-Linux/lib/libfontconfig.so`.

**Comment by stuarteberg** [comment]
My guess is that your OS's version of `libfontconfig.so` is older (or newer) than the version we ship with ilastik, and therefore ilastik's `libfindconfig.so` can't parse your copy of the `/etc/fonts/fonts.conf` file (perhaps the expected syntax of that file has changed).

FWIW, we ship fontconfig 2.11.1 with ilastik.  What version does Ubuntu 14.04 use?

**Comment by dagophil** [comment]
Ubuntu 14.04 ships 2.11.0. I compared my `fonts.conf` with the one from ilastik and there are small differences. However, the syntax seems to be the same:
```
$ diff /etc/fonts/fonts.conf /home/philip/inst/ilastik-1.1.6-Linux/etc/fonts/fonts.conf 
27c27
<       <dir>/usr/local/share/fonts</dir>
---
> 
75c75
<       <cachedir>/var/cache/fontconfig</cachedir>
---
>       <cachedir>/miniconda/envs/ilastik-release/var/cache/fontconfig</cachedir>
```
Replacing `/etc/fonts/fonts.conf` by the ilastik file (or vice versa) did not have an effect.
